examples/Console/TestLogs.cs

```diff
@@ -73,12 +74,36 @@ internal static object Run(LogsOptions options)
                         System.Console.WriteLine($"Export protocol {options.Protocol} is not supported. Default protocol 'grpc' will be used.");
                     }
 
-                    opt.AddOtlpExporter(otlpOptions =>
+                    var processorType = ExportProcessorType.Batch;
+                    if (options.ProcessorType.Trim().ToLower().Equals("batch"))
                     {
-                        otlpOptions.Protocol = protocol;
+                        processorType = ExportProcessorType.Batch;
+                    }
+                    else if (options.Protocol.Trim().ToLower().Equals("simple"))
+                    {
+                        processorType = ExportProcessorType.Simple;
+                    }
+                    else
+                    {
+                        System.Console.WriteLine($"Export processor type {options.ProcessorType} is not supported. Default processor type 'batch' will be used.");
```

Comment:
Skip execution of the remaining code and return?

Reply:
That's the pattern we followed above this. No strong preference, I like this one better as it would export the data atleast.